src/glossary.md

```diff
@@ -65,6 +65,12 @@ For example, `2 + (3 * 4)` is an expression that returns the value 14.
 An [item] that is not a member of an [implementation], such as a *free
 function* or a *free const*. Contrast to an [associated item].
 
+### Inhabited
+
+A type is inhabited if it has constructors and therefore can be instantiated. An inhabited type is
```

Comment:
Is your notion of "has constructors" intended to e.g. take visibility into account? That's the current definition of inhabitedness *in the type system* but *not the abstract machine*. You are describing this from the POV of the type system, right?

Reply:
Uh, yes..? I just borrowed the explanation from [RFC 1216](https://github.com/rust-lang/rfcs/blob/master/text/1216-bang-type.md).

Reply:
I see. :) Well, this is good.